src/coreclr/vm/assembly.cpp

```diff
@@ -1228,8 +1228,7 @@ static void RunMainInternal(Param* pParam)
 
     //<TODO>
     // When we get mainCRTStartup from the C++ then this should be able to go away.</TODO>
```

Comment:
Can you please remove the TODO comment? It has been there even before .NET Core 1.0 and I don't think we will ever do anything with it.